src/odemis/gui/main_xrc.py

```diff
@@ -6797,7 +6798,42 @@ def __init_resources():
                     </object>
                     <flag>wxEXPAND</flag>
                   </object>
-                </object>
+                  <object class="sizeritem">
+					<object class="wxBoxSizer">
+							<object class="sizeritem">
+								<object class="wxStaticText">
+									<label>ZStack Steps</label>
+									<fg>#BFBFBF</fg>
+									<XRCED>
+										<assign_var>1</assign_var>
+									</XRCED>
+									<font>
+										<size>12</size>
+										<encoding>UTF-8</encoding>
+									</font>
+								</object>
+								<flag>wxTOP</flag>
+								<border>25</border>
+							</object>
+							<object class="sizeritem">
+								 <object class="UnitIntegerSlider" name="zstack_steps">
+										<size>-1,20</size>
```

Comment:
It's working fine now but it behaves as `UnitIntegerCtrl` even though in the test case in `comp_slider_test` shows an integer slider.

Reply:
That's odd....